src/routes/projects.routes.js

```diff
@@ -539,6 +539,39 @@ router.get(
   projectController.getBuildStatusesForProject
 );
 
+router.get(
+  '/projects/:projectId/logs/:buildNumber',
+  authController.isAuthenticated,
```

Comment:
This route is missing the api doc.  Also, should this be restricted to just contributors?